xcode/Subconscious/Shared/Components/FirstRun/FirstRunProfileView.swift

```diff
@@ -43,7 +43,7 @@ struct FirstRunProfileView: View {
                 .simultaneousGesture(TapGesture().onEnded {
                     app.send(.createSphere)
                 })
-                .disabled(!app.state.isNicknameTextFieldValid)
+                .disabled(!app.state.nicknameFormField.isValid)
```

Comment:
Nit: consistency, see above. 

Do we prefer `app.state.isNicnameFormField.isValid` or `app.state.isNicknameTextFieldValid`?

Reply:
Fixed